docs/getting_started/examples.rst

```diff
@@ -0,0 +1,66 @@
+Examples and Resources 
+========================
+
+The earlier sections have covered the basics of using Modin to speed up your pandas workflow. Here you can find additional resources to learn about Modin. To learn more about advanced usage for Modin, please refer to :doc:`this section </advanced_usage/index>`.
+
+Usage Examples
+---------------
+
+The following notebooks demonstrate how Modin can be used for scalable data science: 
+
+- Quickstart Guide to Modin [`Source <https://github.com/modin-project/modin/tree/master/examples/quickstart.ipynb>`_]
+- Using Modin with the NYC Taxi Dataset [`Source <https://github.com/modin-project/modin/blob/master/examples/jupyter/NYC_Taxi.ipynb>`_] 
+- Using Experimental Modin in the cloud with the NYC Taxi Dataset on a AWS cluster [`Source <https://github.com/modin-project/modin/blob/master/examples/jupyter/NYC_Taxi_cloud.ipynb>`_] 
+- Modin for Machine Learning with scikit-learn [`Source <https://github.com/modin-project/modin/blob/master/examples/modin-scikit-learn-example.ipynb>`_] 
+
+Tutorials  
```

Comment:
I was not able to get to no one of the exercises. Please check the links.

Reply:
Nice catch! Fixed now!